### PR TITLE
Changed test_file

### PR DESCRIPTION
**Pull request by Karolina-Wardyla**

### CODE REVIEW

File: test_test/test_file.txt [hunk 1]
```diff
@@ -1 +1 @@
-test test test 
\ No newline at end of file
+To jest plik testowy dla Piotrka.
```
Comment (pmzajaczkowski) [inline test_test/test_file.txt]: Proszę o dodanie jeszcze dwóch linii tekstu :grinning:

Comment (Karolina-Wardyla) [reply test_test/test_file.txt]: Ok. No problem.

Comment (Karolina-Wardyla) [reply test_test/test_file.txt]: The task is done.

**Comment by Karolina-Wardyla** [comment]
I have made some changes.

**Pull request merged**